wallet/src/ui/app/pages/home/transfer-coin/CoinSelector.tsx

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import cl from 'classnames';
+import { useMemo, useCallback } from 'react';
+import { useIntl } from 'react-intl';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+
+import { Content } from '_app/shared/bottom-menu-layout';
+import PageTitle from '_app/shared/page-title';
+import Icon, { SuiIcons } from '_components/icon';
+import { useAppSelector } from '_hooks';
+import { accountAggregateBalancesSelector } from '_redux/slices/account';
+import {
+    GAS_TYPE_ARG,
+    SUPPORTED_COINS_LIST,
+} from '_redux/slices/sui-objects/Coin';
+import { balanceFormatOptions } from '_shared/formatting';
+
+import st from './CoinSelector.module.scss';
+
+function CoinsSelectorPage() {
```

Comment:
This component looks very similar with active coin card I think and we have some other components to display coins. Probably fine to add similar code for now but we should try to avoid it if we can

Reply:
This is resolved, combined the two component